lib/x11utils.pm

```diff
@@ -408,6 +409,18 @@ sub turn_off_gnome_screensaver_for_gdm {
     script_run 'sudo -u gdm dbus-launch gsettings set org.gnome.desktop.session idle-delay 0';
 }
 
+=head2 turn_off_gnome_screensaver_for_running_gdm
+
+turn_off_gnome_screensaver_for_running_gdm()
+
+Disable screensaver in gnome for running gdm. The function should be run under root. To be called
+from a command prompt, for example an xterm window.
+
+=cut
+sub turn_off_gnome_screensaver_for_running_gdm {
+    script_run 'su gdm -s /bin/bash -c "DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/$(id -u gdm)/bus gsettings set org.gnome.desktop.session idle-delay 0"';
```

Comment:
How do user know  when to use 'turn_off_gnome_screensaver_for_gdm' when to use 'turn_off_gnome_screensaver_for_running_gdm'?

Reply:
They are different functions, just as function name, one is for running gdm, one is not. As my understanding, running gdm means the gnome started.